Keas.Core/Domain/EquipmentAssignment.cs

```diff
@@ -1,6 +1,6 @@
 namespace Keas.Core.Domain
 {
     public class EquipmentAssignment : AssignmentBase {
-
+        public Equipment Equipment { get; set; }
```

Comment:
@jSylvestre if you are going to add these you should also add the related ID properties, even if you aren't going to use them.  So like add `public int EquipmentId {..}`.